x/blob/README.md

```diff
@@ -46,47 +46,55 @@ celestia-app tx blob payForBlob <hex encoded namespace> <hex encoded data> [flag
 
 ### Programmatic Usage
 
-There are tools to programmatically create, sign, and broadcast `MsgWirePayForBlob`s
+There are tools to programmatically create, sign, and broadcast `BlobTx`s
 
 ```go
-// create the raw WirePayForBlob transaction
-wpfbMsg, err := apptypes.NewWirePayForBlob(block.Header.NamespaceId, blob, 16, 32, 64, 128)
-if err != nil {
-    return err
+func submitPFB() error {
+	blob := []byte{1}
+	namespace := []byte{1, 2, 3, 4, 5, 6, 7, 8}
+
+	// create the raw PayForBlob transaction
+	pfbMsg, err := apptypes.NewPayForBlob(address, namespace, blob)
+	if err != nil {
+		return err
+	}
+
+	// we create a `KeyringSigner` to sign messages programmatically
+	keyringSigner, err := NewKeyringSigner(keyring, "keyring account name", "chain-id-1")
+	if err != nil {
+		return err
+	}
+
+	// query for account information necessary to sign a valid tx
+	err = keyringSigner.QueryAccount(ctx, grpcClientConn)
+	if err != nil {
+		return err
+	}
+
+	// generate the signatures for each `MsgPayForBlob` using the `KeyringSigner`,
+	// then set the gas limit for the tx
+	gasLimOption := types.SetGasLimit(200000)
+
+	// Build and sign the `MsgPayForBlob` tx
+	signedTx, err := keyringSigner.BuildSignedTx(signer.NewTxBuilder(gasLimOption), pfbMsg)
+	if err != nil {
+		return err
+	}
+
+	rawTx, err := signer.EncodeTx(signedTx)
+	if err != nil {
+		return err
+	}
+
+	blobTx, err := coretypes.MarshalBlobTx(rawTx, wblob)
```

Comment:
`wblob` appears undefined

Reply:
incorporated the original suggestion to just remove this blurb of code. Instead, I left links to celestia's nodes submission docs and pointed to the function to reverse engineer for programmatic usage.

7f3148d264c658550c6ad0e76c7561215ec1093e